src/config.rs

```diff
@@ -308,6 +311,12 @@ impl Config {
                 info!("TLS support is disabled");
             }
         };
+        if let Some(true) = self.general.enable_prometheus_exporter {
+            info!(
+                "Exposing prometheus metrics on http://{}:{}/metrics",
```

Comment:
This will print on every config reload I believe. Could you move that into `main.rs` when it launches the http listener?